### DISCUSSION TITLE
Making it easier

### DISCUSSION THREAD

**Discussion by microprediction** [microprediction/microprediction#65] [Ideas]
Ideas for making it easier

**Comment by tdaula** [comment]
Big picture from my amateur seat for general populace end-user adoption - Jupyter notebook (or other online environment) back end with a bot front-end to walk them through setting up the parameters ("is your data like A, B, or C?",  "when will you publish?", etc.). You'd need a companion page on what makes a good dataset and realistic expectations (i.e., good objective functions and what you can hope to achieve). Perhaps a comparison (cost, appropriateness, pitfalls) to other black box products applicable to other types of problems (Prophet, Data Robot).
